### ISSUE TITLE
CWE for node-forge

### ISSUE THREAD

**Issue by runthis** [laravel-mix/laravel-mix#3199]
-   Laravel Mix Version: 6.0.39
-   Node Version: 14.17.2
-   NPM Version: 8.1.4
-   OS: Ubuntu

### Description:
https://github.com/advisories/GHSA-5rrq-pxf6-6jx5

Probably already aware, but 4 low severity vulnerability in node-forge <1.0.0. Downgrading to mix <6.0.19 introduces 30 or so medium severity due to postcss.

```# npm audit report

node-forge  <1.0.0
Prototype Pollution in node-forge debug API. - https://github.com/advisories/GHSA-5rrq-pxf6-6jx5
fix available via `npm audit fix --force`
Will install laravel-mix@6.0.18, which is a breaking change
node_modules/node-forge
  selfsigned  1.1.1 - 1.10.14
  Depends on vulnerable versions of node-forge
  node_modules/selfsigned
    webpack-dev-server  2.5.0 - 4.7.2
    Depends on vulnerable versions of selfsigned
    node_modules/webpack-dev-server
      laravel-mix  1.2.0 - 6.0.0-beta.17 || >=6.0.19
      Depends on vulnerable versions of webpack-dev-server
      node_modules/laravel-mix

4 low severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force```


### Steps To Reproduce:
Install laravel-mix >=6.0.19

**Comment by HDVinnie** [comment]
Also getting this.

**Comment by hebbet** [comment]
see #3197 and #3198

**Comment by thecrypticace** [comment]
Published 6.0.40 to deal with this. Sorry for the delay!
